Library/Formula/wifi-password.rb

```diff
@@ -0,0 +1,14 @@
+class WifiPassword < Formula
+  homepage "https://github.com/rauchg/wifi-password"
+  url "https://github.com/rauchg/wifi-password/archive/0.0.1.tar.gz"
+  version "0.0.1"
```

Comment:
`version` is redundant with the version extracted from the tarball filename so you can remove it